### PR TITLE
add scripts to clean experimental rpms and debs

### PR DESCRIPTION
**Pull request by jessfraz**
ping @tianon @mikedougherty @calavera @tiborvass

### CODE REVIEW

File: hack/make/clean-apt-repo [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# This script cleans the experimental pool for the apt repo.
+# This is useful when there are a lot of old experimental debs and you only want to keep the most recent.
+#
+
+: ${DOCKER_RELEASE_DIR:=$DEST}
+APTDIR=$DOCKER_RELEASE_DIR/apt/repo/pool/experimental
+: ${DOCKER_ARCHIVE_DIR:=$DEST/archive}
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+latest_versions=$(dpkg-scanpackages "$APTDIR" /dev/null 2>/dev/null | grep Filename | awk '{print $2}')
```
Comment (tianon) [inline hack/make/clean-apt-repo]: `awk -F ': ' '$1 == "Filename" { print $2 }'` (no need for `grep`)

**Comment by tianon** [comment]
Seems generally sane :+1:

File: hack/make/clean-apt-repo (hunk 1 above)
Comment (jfrazelle) [inline hack/make/clean-apt-repo]: niceeee

**Comment by mikedougherty** [comment]
sgtm

**Comment by tiborvass** [comment]
thanks Jess!

**Comment by thaJeztah** [comment]
@tianon is that a "lgtm", or a "I need another look"?

**Comment by calavera** [comment]
LGTM

**Comment by tianon** [comment]
@thaJeztah sorry, that was a "I don't deploy this code as often as Jess, so I trust her judgement and the code looks sane"

LGTM :+1:

**Comment by thaJeztah** [comment]
> "I don't deploy this code as often as Jess, so I trust her judgement and the code looks sane"

@tianon bbbuttt you're our Bash king, so having a LGTM from you makes me comfortable, knowing that "scripting looks solid"

**Comment by jfrazelle** [comment]
Tianon grew up with bash it was forced on his as a child and has morphed him into the bash king today. Waiting for the Marvel movie to come out now.

**Comment by thaJeztah** [comment]
"Bashman" the movie!

Let's call a city after him

Done!

https://en.wikipedia.org/wiki/Bashman

**Comment by tianon** [comment]
I'd so watch that.

**Pull request merged**